test/functional/wallet_avoid_mixing_output_types.py

```diff
@@ -124,6 +124,7 @@ def set_test_params(self):
 
     def skip_test_if_missing_module(self):
         self.skip_if_no_wallet()
+        self.skip_if_no_sqlite()
```

Comment:
this doesn't seem to fix it :/
```
python test/functional/wallet_avoid_mixing_output_types.py
...
test_framework.authproxy.JSONRPCException: Legacy wallets cannot provide bech32m addresses (-8)
```

Reply:
i think this is the best we can do right now with the test framework. if no sqlite, skip. if sqlite and bdb, pass the `--descriptors` flag, else the test will fail